lib/game_io.ex

```diff
@@ -38,12 +51,24 @@ defmodule GameIO do
     end
   end
 
-  def invalid_input(msg) do
-    IO.puts(msg)
+  def player_selection() do
+    get_player_selection |> parse_input
+  end
+
+  def game_mode_selection() do
+    get_game_mode_selection() |> parse_input
+  end
+
+  def invalid_move(move, msg) do
+    IO.puts("'#{move}' is not #{msg}. Please try again.")
+  end
+
+  def invalid_selection(selection, subject) do
+    IO.puts("\n#{selection} is not a valid #{subject} choice. Please try again.")
   end
 
-  def print_result(board) do
-    if Game.check_draw(board), do: IO.puts("Cat's Game!"), else: IO.puts("#{Game.winner(board)} won!")
+  def print_result(winner) do
```

Comment:
You can define multiple print_results depending on true/false/nil instead of having the if inside it